hal/network/ncp_client/realtek/rtl_ncp_client.cpp

```diff
@@ -286,13 +287,48 @@ int RealtekNcpClient::connect(const char* ssid, const MacAddress& bssid, WifiSec
 }
 
 int RealtekNcpClient::getNetworkInfo(WifiNetworkInfo* info) {
+    int rtlError = 0;
```

Comment:
Just in case run `wiring/no_fixture_wifi`.

Reply:
👍 tests passed